### ISSUE TITLE
Monitoring - some graphs are missing in RKE cluster  

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#25201]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
- run Rancher v2.3.5 or master-head
- add a DO cluster
```
K8s: v1.17.2
OS of nodes: Ubuntu 16.04.6 LTS
```
- enable the cluster monitoring (0.0.7)
- check requests sent to Rancher server for metrics data

**Result:**

- one new graph is added to the section `etcd metrics`
```
etcd-peer-traffic
```

- the following 3 metrics are missing in the section `Kubernetes component metrics`
```
    "apiserver-request-count", 
    "scheduler-e-2-e-scheduling-latency-seconds-quantile", 
    "apiserver-request-latency",
```

**Comment by jiaqiluo** [comment]
This issue is also seen in a custom RKE cluster using SUSE
```
k8s: v1.17.2 
OS: SUSE Linux Enterprise Server 12 SP5
```
 
-  the following 3 metrics are missing in the section `Kubernetes component metrics`
```
 "apiserver-request-count"
 "scheduler-e-2-e-scheduling-latency-seconds-quantile"
 "apiserver-request-latency"
```

**Comment by daxmc99** [comment]
https://kubernetes.io/docs/setup/release/notes/#deprecated-changed-metrics
Some of the metrics used in these graphs has been turned off in k8s 1.17. 
`apiserver-request-count` & `apiserver-request-latency`

**Comment by deniseschannon** [comment]
Based on @daxmc99's comment, it seems as if these graphs are intentionally no longer available with k8s 1.17
